repose-aggregator/components/filters/client-auth/src/test/groovy/org/openrepose/filters/clientauth/AuthNFilterDeprecationTest.groovy

```diff
@@ -0,0 +1,32 @@
+package org.openrepose.filters.clientauth
+
+import com.mockrunner.mock.web.MockFilterConfig
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.LoggerContext
+import org.apache.logging.log4j.test.appender.ListAppender
+import org.openrepose.core.services.config.ConfigurationService
+import org.openrepose.core.services.datastore.DatastoreService
+import spock.lang.Specification
+
+import static org.mockito.Mockito.mock
+
+class AuthNFilterDeprecationTest extends Specification {
+    def 'logs a deprecation warning upon init'() {
+        given:
+        System.setProperty("javax.xml.parsers.DocumentBuilderFactory",
```

Comment:
Depend on different libraries?

I checked the full maven dependency tree and only noted a few oddities with regards to Xerces:
1. The translation module gets a transitive xerces pulled in. It's coming from some xmlcalabash library, a test dependency. This probably isn't what we want, but the behavior should not be affected by any on the changes I've made.
2. wadl-tools pulls in an older version of the Rackspace xerces. Again, this shouldn't be affected by the changes I made, but it should probably get excluded.